yarn.lock

```diff
@@ -5412,6 +5412,19 @@ convert-source-map@^2.0.0:
   resolved "https://registry.yarnpkg.com/convert-source-map/-/convert-source-map-2.0.0.tgz#4b560f649fc4e918dd0ab75cf4961e8bc882d82a"
   integrity sha512-Kvp459HrV2FEJ1CAsi1Ku+MY3kasH19TFykTz2xWmMeq6bk2NU3XXvfJ+Q61m0xktWwt+1HSYf3JZsTms3aRJg==
 
+copyfiles@2.4.1:
```

Comment:
I used cpx in the process-editor-client, but this is also not really maintained. 
But there is a fork cpx2 which is. So maybe we should change that in the future, but definitely ok for now.